prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts

```diff
@@ -63,6 +64,13 @@ export class CollegeCertificationFormComponent implements OnInit {
 
   public licenseGrouping = LicenseGrouping;
 
+  /**
+   * 21 - College of Health and Care Professionals of BC
+   * 22 - College of Complementary Health Professionals of BC
+   */
+  public amalgamatedColleges: number[] =
```

Comment:
I think need to be more specific:  `allowDupAmalgamatedColleges`

Reply:
Updated